### PR TITLE
Fix/16483 progress bar was broken

### PR DESCRIPTION
**Pull request by tessi**
- re-implements the progress bar to
  *\* not use a table-layout (in angular as well as in the non-angular pages)
  *\* not have box-elements inside of inline-elements in the HTML
- fixes: https://www.openproject.org/work_packages/16483
- documents `progress_bar` helper
- fixes flaws in the roadmap view
  *\* the percent-sign was drawn twice in the progress bar legend
  *\* `start_date` and `due_date` are only printed now when they have a value (so we have no empty labels confusing the user)
- renames occurences of the word `pourcent` (which is frensh) to 'percent'

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://fix-16483-progress-bar-was-broken.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

**Comment by tessi** [comment]
FYI: I know the tests are failing atm. I'm currently working on that. Don't hesitate to review, though. The tests fail only because I changed the HTML layout. The progress_bar itself should work properly.

File: app/assets/stylesheets/legacy/_06_progress_bar.sass [hunk 1]
```diff
@@ -28,33 +28,25 @@
 
 /***** Progress bar ****
```
Comment (myabc) [inline app/assets/stylesheets/legacy/_06_progress_bar.sass:30]: @tessi can you move this out of this file and under `app/assets/stylesheets/content` instead?

Comment (tessi) [inline app/assets/stylesheets/legacy/_06_progress_bar.sass:30]: sure!

**Pull request merged**